### PR TITLE
Add contact from in the contact section

### PR DESCRIPTION
**Pull request by akhror-valiev**
- Used flex
- Added form tag to the contact section
- Added style to the contact section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -208,5 +204,53 @@ <h3>LET'S CONNECT</h3>
...
+              method="post"
+            >
+              <fieldset id="fs-frm-inputs">
+                <input
+                  type="text"
+                  name="name"
+                  id="full-name"
+                  required=""
+                  placeholder="Oshie"
+                />
```
Comment (roshan-bajgain) [inline index.html]: - [ ] According to project requirements, the name field should accept up to 30 chars. please implement it.

File: index.html [hunk 2]
```diff
@@ -208,5 +204,53 @@ <h3>LET'S CONNECT</h3>
...
+                  required=""
+                  placeholder="oshie0115@gmail.com"
+                />
+
+                <textarea
+                  rows="5"
+                  name="message"
+                  id="message"
+                  placeholder="Write your message"
+                  required=""
```
Comment (roshan-bajgain) [inline index.html]: - [ ] According to project requirements, the Message field should accept up to 500 chars. please implement it.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @akhror-valiev and @SekoViper, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -222,15 +222,16 @@ <h2>Contact Me</h2>
                   type="text"
                   name="name"
                   id="full-name"
-                  required=""
+                  required
```
Comment (youmari) [inline index.html:225]: - [ ] [ OPTIONAL] it would be better if you use the right font-size for the all input in order to match the Figma design

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @akhror-valiev 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (akhror-valiev) [reply index.html:225]: @youmari thanks, noted!

**Pull request merged**